palmid.Rmd

```diff
@@ -310,11 +318,13 @@ plotly::ggplotly(date.report)
 
 #### Virus-Organism Associations
 
-Organism meta-data from the SRA sequencing libraries matching the input-virus.
-Word size and color is scaled by proximity of the input-virus to its match in 
-the library. Wordclouds show a) Researcher provided annotations of the libraries
-or b) STAT automated taxonomic (orders) analysis of the reads contained within
-the library ([Katz et al., 2021](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02490-0)).
+Organism meta-data from the SRA sequencing libraries matching the
+input-virus. Word size and color is scaled by proximity of the
+input-virus to its match in the library. Wordclouds show a) Researcher
+provided annotations of the libraries or b) STAT automated taxonomic
+(orders) analysis of the reads contained within the library ([Katz et
```

Comment:
Weird line break here? Did you run this through a linter?

Reply:
I didn't run through a linter actually. Just pushed fixes to weird line breaks.